docs/fiddles/system/system-app-user-information/app-information/renderer.js

```diff
@@ -0,0 +1,23 @@
+const {ipcRenderer} = require('electron')
+
+const appInfoBtn = document.getElementById('app-info')
+const links = document.querySelectorAll('a[href]')
+
+appInfoBtn.addEventListener('click', () => {
+  ipcRenderer.send('get-app-path')
+})
+
+ipcRenderer.on('got-app-path', (event, path) => {
+  const message = `This app is located at: ${path}`
+  document.getElementById('got-app-info').innerHTML = message
+})
+
+Array.prototype.forEach.call(links, (link) => {
```

Comment:
Instead of an array here, could you add a unique ID and use `document.getElementById` to target the specific link and use `addEventListener` on it? Can get rid of the loop because you're  targeting a single `<a>`